src/ResourceManagement/SiteRecovery/SiteRecoveryManagement/Microsoft.Azure.Management.RecoveryServicesARM.nuget.proj

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <!--
+    Microsoft.Azure.Management.RecoveryServicesARM
+    -->
+    <SdkNuGetPackage Include="Microsoft.Azure.Management.RecoveryServicesARM">
```

Comment:
please remove ARM suffix from the package name in the nuget.proj and nuspec files and their names.